src/pages/CollectionPage/index.tsx

```diff
@@ -0,0 +1,44 @@
+import { useEffect, useState } from "react"
+import { Container, Row, Col, Card } from 'react-bootstrap';
+import { Collection } from "@imtbl/core-sdk";
+import HeaderBar from "../../components/HeaderBar/HeaderBar"
+import { getCollections } from "../../api/collections-api";
+import { Link } from "react-router-dom";
+import placeholderImg from '../../asset/placehoderImg.jpg';
+
+const CollectionsPage = () => {
+    const [result, setResult] = useState<Collection[]>([])
```

Comment:
useState() over useState([]), this forces you to handle undefined/nil case

I could be wrong here frontend devs please correct me :)

Reply:
I need an empty array for the first load. While retrieving the data, the browser renders 'Loading...' on the first load.